draft-irtf-cfrg-opaque.md

```diff
@@ -1665,6 +1663,64 @@ learn the client's password. This means that additional checks should be conside
 a production deployment of OPAQUE: for instance, ensuring that there is a
 server-authenticated channel over which OPAQUE registration and login is run.
 
+# API Considerations
+
+## Errors {#api-errors}
+
+Some functions included in this specification are fallible. For example,
+MACs exchanged during the AKE might fail verification, yielding an error. Additionally, protocol messages 
+sent between client and server might not match their expected size, yielding a different error.
+The following errors are documented for the OPRF-3DH composition in this specification and the recommended
+configurations, and do not cover potential errors in other configurations. Participants MUST abort the
+protocol when these errors occur.
+
+### Deserialization errors {#deserialization-errors}
+
+| Name                                 | Role   | Messages                  | Reason                                               |
+|:-------------------------------------|:-------|:--------------------------|:-----------------------------------------------------|
+| InvalidMessageLengthError            | Both   | All                       | The message length is invalid for the configuration. |
+| InvalidBlindedDataError              | Server | RegistrationRequest, Ke1  | Blinded data is an invalid point.                    |
```

Comment:
```suggestion
| InvalidBlindedDataError              | Server | RegistrationRequest, Ke1  | Blinded data is an invalid element.                    |
```

Reply:
Is this raised in the spec? I guess it's implicit, but we should probably note that otherwise.

Reply:
What do you mean?

Reply:
Yes, it's implicit. How do you suggest we should specify that?

Reply:
Add a second table with the implicit exceptions that come from other specs?